### PR TITLE
Fixes overlapping elements in the editor

### LINKED ISSUE #20837: Some elements of the editor overlap with others
**Godot version:**
3.1 9bd5315

**OS/device including version:**
Windows 10 Intel HD 4600

**Screen Resolution**
Around 1920x540px

**Issue description:**
When height of editor is small, some elements from right panel  overlap others, part around Node and Inspector tab are transparent and that do not look good.

I checked this and this seems to be caused by Profiler, other tabs, like Errors and Debugger looks good until I click to Profiler

https://streamable.com/mc97n

### PR DESCRIPTION
**Pull request by tomasantunes**
_Bugsquad edit: fixes #20837_

### CODE REVIEW

File: editor/editor_node.cpp [hunk 1]
```diff
@@ -4839,6 +4839,7 @@ EditorNode::EditorNode() {
 	VBoxContainer *center_vb = memnew(VBoxContainer);
 	main_hsplit->add_child(center_vb);
 	center_vb->set_h_size_flags(Control::SIZE_EXPAND_FILL);
+	center_vb->set_custom_minimum_size(Size2(700, 0) * EDSCALE);
```
Comment (akien-mga) [inline editor/editor_node.cpp]: I don't think this is the proper fix, that's too arbitrary.

Comment (tomasantunes) [reply editor/editor_node.cpp]: It's the width of the profiler. What size do you recommend?

**Comment by akien-mga** [comment]
Moving to 3.2 milestone as we're about to enter the beta stage and *release freeze* for Godot 3.1. Only bug fixes against issues of the 3.1 milestone (or enhancements to core features of the 3.1 roadmap) will be considered until 3.1-stable is released.

Note: If it's merged after 3.1-stable is released, this change may be considered for cherry-picking into a later 3.1.x release.

**Comment by akien-mga** [comment]
The original issue has been fixed in the master branch, so this change is no longer necessary. Thanks for the contribution nevertheless!

**Pull request closed without merging**